### ISSUE TITLE
Hex support

### ISSUE THREAD

**Issue by vintikzzz** [matehat/cqerl#124]
Hi! Could you be so kind to add cqerl to [Hex](https://hex.pm/). [Guide](https://www.rebar3.org/docs/publishing-packages) is already there. Currently i am trying to publish another package that depends on cqerl and got this:
```
** (Mix) Stopping package build due to errors.
Dependencies excluded from the package (only Hex packages can be dependencies): cqerl
```

**Comment by matehat** [comment]
Got the same problem myself! Some of this package's dependencies aren't hex packages.

Well, let me try again now to see if they moved to hex.

**Comment by bettio** [comment]
Is anything changed so far? which are the dependencies that are still missing?

**Comment by matehat** [comment]
Just landed in https://github.com/matehat/cqerl/commit/839132cbe6c786cedb4400876c9006b26b6a662f
